lib/tasks/auth_source_external.rake

```diff
@@ -0,0 +1,8 @@
+namespace :auth_source_external do
+  desc 'Creates an AuthSourceExternal by name "External", if no AuthSourceExternal exists.'
+  task :create => :environment do
+    User.as_anonymous_admin do
+      AuthSourceExternal.create!(name: 'External') unless AuthSourceExternal.any?
```

Comment:
If an external auth source already exists we should probably print out a message letting the user know it's name and ID so they can edit it if needed

Reply:
makes sense, will add that right away!